### DISCUSSION TITLE
If I enable “Hide uYou tab” in uYou settings, how do I get it back?

### DISCUSSION THREAD

**Discussion by WoodpeckerBaby** [arichornlover/uYouEnhanced#143] [Q&A]
Except reinstalling the app.

**Comment by arichornlover** [comment]
That’s a good question. Go to General in Settings and the first thing you see is probably `uYou`. once you see tap on it and it will take you to the uYou settings.

**Comment by arichornlover** [comment]
Update: if you are using uYouEnhanced v19.03.2-3.0.1 or later then you cannot get it back after hiding it and it is the truth. So it is recommended to use uYouEnhanced v19.02.1-3.0.1.

**Reply by yodaluca23** [reply]
> Update: if you are using uYouEnhanced v19.03.2-3.0.1 or later then you cannot get it back after hiding it and it is the truth. So it is recommended to use uYouEnhanced v19.02.1-3.0.1.

Do you know if MiRO92 is planning on making a new update soon to fix it or could you like add a button that just calls whatever is called to pull up the menu.., (idrk how swift works)

**Reply by arichornlover** [reply]
That could be possible. But I don’t know the controller that calls the uYou Menu, if I known that then I could’ve made a uYou Fix tweak to make it work.
